pycbc/waveform/waveform.py

```diff
@@ -110,6 +110,8 @@ def _check_lal_pars(p):
         lalsimulation.SimInspiralWaveformParamsInsertFrameAxis(lal_pars, p['frame_axis'])
     if p['side_bands']:
         lalsimulation.SimInspiralWaveformParamsInsertSideband(lal_pars, p['side_bands'])
+    if p['mode_array']:
```

Comment:
I agree with Ian that I don't know if I agree or disagree :)
But the version you suggested Collin is now up